--- oracle
+++ port
@@ -30 +30 @@
-0030 parse:                         <bind \x5C^X ed-start-over> tok=0 argc=3 parse=0
+0030 parse:                         <bind \x5C^X ed-start-over> tok=0 argc=3 parse=1
@@ -32,3 +32,3 @@
-0032 parse:                         <bind -a ^A ed-move-to-end> tok=0 argc=4 parse=0
-0033 parse:                         <bind ^A> tok=0 argc=2 parse=0
-0034 parse:                         <bind -l> tok=0 argc=2 parse=0
+0032 parse:                         <bind -a ^A ed-move-to-end> tok=0 argc=4 parse=1
+0033 parse:                         <bind ^A> tok=0 argc=2 parse=1
+0034 parse:                         <bind -l> tok=0 argc=2 parse=1
@@ -44,2 +44,2 @@
-0044 parse:                         <history size 42> tok=0 argc=3 parse=1
-0045 parse:                         <history unique 1> tok=0 argc=3 parse=1
+0044 parse:                         <history size 42> tok=0 argc=3 parse=0
+0045 parse:                         <history unique 1> tok=0 argc=3 parse=0
@@ -47 +47 @@
-0047 parse:                         <bind> tok=0 argc=1 parse=0
+0047 parse:                         <bind> tok=0 argc=1 parse=1
@@ -49 +49 @@
-0049 parse:                         <bind -q ^A ed-move-to-beg> tok=0 argc=4 parse=0
+0049 parse:                         <bind -q ^A ed-move-to-beg> tok=0 argc=4 parse=1
@@ -53,3 +53,3 @@
-0053 parse:                         <history> tok=0 argc=1 parse=1
-0054 parse:                         <history size> tok=0 argc=2 parse=1
-0055 parse:                         <history size not-a-number> tok=0 argc=3 parse=1
+0053 parse:                         <history> tok=0 argc=1 parse=0
+0054 parse:                         <history size> tok=0 argc=2 parse=0
+0055 parse:                         <history size not-a-number> tok=0 argc=3 parse=0
